latex/chain.tex

```diff
@@ -1222,14 +1215,14 @@ \subsection{Chain Transition}
         \left(
           {\begin{array}{c}
               \var{b} \\
-              \var{ru}' \\
+              \var{ru}' \\  % TODO: Which variable here? es/es' ?
```

Comment:
good question/catch!  it actually doesn't matter (I'll explain) but I think that `es'` is probably clearer and more consistent.

the rule `RUPD` will not fire on the first slot of a new epoch because of the check that `s > ( firstSlot epoch s ) + StartRewards`.  (Well, technicality, I should say that equation 29 does not fire, but instead equation 31 fires, which is a no-op).  Therefore `es` and `es'` are the same when `RUPD` changes `ru`.